script/release-workflow/docker-push.sh

```diff
@@ -1,15 +1,29 @@
 #!/bin/sh
 
-set -euo >/dev/null
+set -eo >/dev/null
```

Comment:
I am not sure of the implications of this really 😓 I needed to do it for _something_

Reply:
The `u` should be in there. It means that you've not set an environment variable that should be set.

See https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/  and scroll to the `set -u` section.

Reply:
ty 👍🏾 added back in.